### PR TITLE
fix: Safari 16 adoptNode issue

### PR DESCRIPTION
**Pull request by mahwy**
# Pull Request

## 📖 Description

Safari 16+ introduced a breaking change with `adoptNode`. It's providing a fix by calling `importNode` in case the adoptNode returned [preemptively](https://github.com/web-platform-tests/wpt/pull/16348) without adopting a given fragment.

### 🎫 Issues

- https://github.com/microsoft/fast/issues/6537
- https://github.com/aurelia/framework/issues/1003

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by mahwy** [comment]
@microsoft-github-policy-service agree company="Fevo"

**Comment by mahwy** [comment]
> @mahwy please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement


@microsoft-github-policy-service agree company="Fevo"

**Review by eljefe223 — APPROVED** [review]

File: change/@microsoft-fast-element-6cd8429d-8362-443b-b70d-b125b2146c1b.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "type": "patch",
+    "comment": "Fix Safari 16 adoptNode issue",
+    "packageName": "@microsoft/fast-element",
+    "email": "mahwy.asdf@gmail.com",
+    "dependentChangeType": "patch"
```
Comment (nicholasrice) [inline change/@microsoft-fast-element-6cd8429d-8362-443b-b70d-b125b2146c1b.json:6]:
```suggestion
    "dependentChangeType": "prerelease"
```

File: change/@microsoft-fast-element-6cd8429d-8362-443b-b70d-b125b2146c1b.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "type": "patch",
```
Comment (nicholasrice) [inline change/@microsoft-fast-element-6cd8429d-8362-443b-b70d-b125b2146c1b.json:2]:
```suggestion
    "type": "prerelease",
```

**Comment by janechu** [comment]
This appears to have been fixed by Safari via https://github.com/WebKit/WebKit/pull/11347

**Pull request closed without merging**